drivers/unix/ip_unix.cpp

```diff
@@ -34,8 +34,8 @@
 
 #ifdef WINDOWS_ENABLED
 #include <stdio.h>
-#include <windows.h>
 #include <winsock2.h>
+#include <windows.h>
```

Comment:
You should add a newline between the two (and ideally a comment explaining that windows.h must be included after winsocks2.h, otherwise clang-format will reorder them again (it orders includes alphabetically by blocks).